frontend/src/constants/policySummary.js

```diff
@@ -1,6 +1,6 @@
 export const POLICY_SUMMARY = (
   <>
-    우아한형제들은 개인정보보호법, 정보통신망 이용촉진 및 정보보호 등에 관한 법률 등 관련 법령 상의
+    우아한형제들은 개인정보 보호법, 정보통신망 이용촉진 및 정보보호 등에 관한 법률 등 관련 법령상의
```

Comment:
회사에서 받은 문구라 그대로 두는 게 좋을 것 같아요.

```suggestion
    우아한형제들은 개인정보보호법, 정보통신망 이용촉진 및 정보보호 등에 관한 법률 등 관련 법령 상의
```